### PR TITLE
CnC on ICP: Updates for 1.0.5 on 8/2

### PR DESCRIPTION
**Pull request by scot-taylor-ibm**
Production push for 8/2 release.

### CODE REVIEW

File: deploy.md [hunk 1]
```diff
@@ -100,94 +100,88 @@ Download the `IBM_WTSN_COMPARE_AN_COMPL_ELEM_CL.tar.gz` package file from [Passp
...
+  The extracted file includes a number of directories and files, including a top-level directory named `ibm-watson-compare-comply-prod` that contains a directory named `templates`.
+
   1. Change to the `ibm-watson-compare-comply-prod/templates` directory:
   
- 	```bash
- 	cd ibm-watson-compare-comply-prod/templates
- 	```
- 	{: pre}
-
+  ```bash
```
Comment (NickyBleiel) [inline deploy.md]: The new info is not indented the same way as the info it replaces. Will it display properly in the numbered list?

File: deploy.md [hunk 2]
```diff
@@ -100,94 +100,88 @@ Download the `IBM_WTSN_COMPARE_AN_COMPL_ELEM_CL.tar.gz` package file from [Passp
...
- 	
- 	**Updated**
- 	```
- 	- name: {{ .Chart.Name }}
- 		image: "mycluster.icp:8500/default/cnc-frontend:v2.3.12"
- 		imagePullPolicy: Always
- 	```
- 	Save and close the `deployment.yaml` file.
- 	
+  **Original**
```
Comment (NickyBleiel) [inline deploy.md]: Same comment as above.

File: logging.md [hunk 3]
```diff
@@ -18,77 +18,14 @@ lastupdated: "2018-03-23"
...
-
-    0 directories, 9 files
-    ```
+{: #logging}
 
 ## Importing the logging dashboards
 
 To import the logging dashboards for {{site.data.keyword.cnc_short}} into IBM Cloud Private, perform the following steps.
 
+  1. Ensure that you have extracted and generated the logging dashboards as described in [Step 1: Download, extract, and render the dashboard templates](monitor.html#monitor).
```
Comment (NickyBleiel) [inline logging.md]: Link does not include the full path.

File: metrics.md [hunk 4]
```diff
@@ -18,81 +18,16 @@ lastupdated: "2018-06-28"
...
-
-   0 directories, 9 files
-   ```
+You can monitor the status of {{site.data.keyword.cnc_short}} by using IBM Cloud Private's monitoring dashboard. The monitoring dashboard uses Grafana for metrics, Prometheus for alerts, and Kibana for logging to present detailed information about your {{site.data.keyword.cnc_short}} instance.
 
 ## Importing the metrics dashboard
 
 To import the metrics dashboard for {{site.data.keyword.cnc_short}} into IBM Cloud Private, perform the following steps.
 
+  1. Ensure that you have extracted and generated the metrics dashboards as described in [Step 1: Download, extract, and render the dashboard templates](monitor.html#monitor).
```
Comment (NickyBleiel) [inline metrics.md]: path

File: logging.md (hunk 3 above)
Comment (scot-taylor-ibm) [reply logging.md]: Fixed

**Pull request merged**